packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1836,23 +1836,29 @@ export default class Meeting extends StatelessWebexPlugin {
 
     LoggerProxy.logger.info(`${LOG_HEADER} Muting Audio`);
 
-    return this.audio.toggle({
+    return logRequest(this.audio.toggle({
```

Comment:
This specific method was not using the `logRequest()` method like the other three.